StripeFinancialConnections/StripeFinancialConnections/Source/Native/NativeFlowController.swift

```diff
@@ -1066,8 +1066,14 @@ private func CreatePaneViewController(
             .analyticsClient
             .log(
                 eventName: "pane.launched",
+                parameters: {
+                    var parameters: [String: Any] = [:]
+                    parameters["referrer_pane"] = dataManager.lastPaneLaunched?.rawValue
```

Comment:
[Grep of Stripe.js parameter to check for consistency](https://livegrep.corp.stripe.com/search/stripe?q=referrer_pane&fold_case=auto&regex=false&context=true)